lib/utility/convertDateToJSFDateTime.js

```diff
@@ -2,8 +2,8 @@ var jsforce = require('jsforce');
 
 /**
  * Convert Date filter props to SalesForce based date fields
- * @param {field} value
- * @param {fieldType} salesforce type value
+ * @param {field} Date in a string format
+ * @param {fieldType} Original type from salesforce schema
```

Comment:
This is still wrong. The curly braces should contain the type, not the param name. So this should be `@param {string} fieldType Original type from salesforce schema`. Same with the first param. See http://usejsdoc.org/tags-param.html 😉